uk.co.piggz.amazfish.json

```diff
@@ -0,0 +1,151 @@
+{
+  "app-id": "uk.co.piggz.amazfish",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15-23.08",
+  "sdk": "org.kde.Sdk",
+  "command": "run-amazfish.sh",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--device=dri",
+    "--system-talk-name=org.bluez",
+    "--talk-name=org.mpris.MediaPlayer2.*",
+    "--talk-name=io.github.rinigus.PureMaps",
+    "--talk-name=org.freedesktop.Notifications"
+  ],
+  "modules": [
+    {
+      "name": "nemo-qml-plugin-dbus",
+      "buildsystem": "qmake",
+      "builddir": true,
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/sailfishos/nemo-qml-plugin-dbus/archive/refs/tags/2.1.33.tar.gz",
+          "sha256": "08d1bdc5eac5eba2197f2c1e04a7e851437724331215059db1d68828dbf3b0a8"
+        },
+        {
+          "type": "patch",
+          "path": "nemodbus-qmake.patch"
+        }
+      ]
+    },
+    {
+      "name": "qtmpris",
+      "buildsystem": "qmake",
+      "builddir": true,
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/sailfishos/qtmpris/archive/refs/tags/1.0.6.tar.gz",
+          "sha256": "384eac76ec7d29d1de6fa93c45007ccd506d6cc0e4c69e19d4250e16cc0f34a2"
+        },
+        {
+          "type": "patch",
+          "path": "qtmpris-qmake.patch"
+        }
+      ]
+    },
+    {
+      "name": "libkdb3",
+      "buildsystem": "cmake",
+      "builddir": true,
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://invent.kde.org/libraries/kdb/-/archive/tags/v3.2.0/kdb-v3.2.0.tar.gz",
+          "sha256": "3a5c660f0080c6971ac1046be84a0accd86a5296e9df4c585b1b91361b5eed20"
+        },
+        {
+          "type": "patch",
+          "path": "6.patch"
+        },
+        {
+          "type": "patch",
+          "path": "upstream_Fix-build-with-newer-Qt.patch"
+        },
+        {
+          "type": "patch",
+          "path": "libkdb3-features.patch"
+        }
+      ]
+    },
+    {
+      "name": "extra-cmake-modules",
+      "buildsystem": "cmake-ninja",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://invent.kde.org/frameworks/extra-cmake-modules/-/archive/v5.115.0/extra-cmake-modules-v5.155.0.tar.gz",
```

Comment:
isn't that in the runtime already?

Reply:
I wasn't able to compile bluez-qt without that.

Reply:
at last and a `cleanup` with the rule `*` to not include it in the final package.

Reply:
cleanup added